### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by ljubar**
| Q                         | A
| ------------------------- | ---
| Bug fix?                  | yes
| New feature?              | no
| Fixed tickets             | SWP-848
| License                   | AGPLv3

### CODE REVIEW

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js [hunk 1]
```diff
@@ -44,30 +44,43 @@ $(document).ready(function() {
...
             $inputs.each(function() {
                 values[this.name] = $(this).val();
               });
 
           var username = values["_username"];
           var password = values["_password"];
-          var user;
-          var apiKey;
+          var data = {'auth': {'username': username, 'password': password}},
+              url = "/app_dev.php/api/v1/auth/";
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: remove `app_dev.php`

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js [hunk 2]
```diff
@@ -107,48 +118,50 @@ $(document).ready(function() {
...
+        var authToken = localStorage.getItem('auth_token');
+        var userId = localStorage.getItem('user_id');
+        if (authToken) {
+          $.ajax({
+            headers : {
+              'Accept' : 'application/json',
+              'Content-Type' : 'application/json',
+              'Authorization' : authToken
+            },
+            url : "/app_dev.php/api/v1/users/profile/" + userId,
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: remove `app_dev.php/`

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js [hunk 3]
```diff
@@ -166,9 +179,9 @@ $(document).ready(function() {
                     headers : {
                         'Accept' : 'application/json',
                         'Content-Type' : 'application/json',
-                        'Authorization' : apiKey
+                        'Authorization' : authToken
                     },
-                    url : "/app_dev.php/api/v1/users/profile/" + user["id"],
+                    url : "/app_dev.php/api/v1/users/profile/" + userId,
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: remove `app_dev.php/`

Comment (ljubar) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: @ahilles107  removed

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js [hunk 4]
```diff
@@ -44,30 +44,43 @@ $(document).ready(function() {
...
           var password = values["_password"];
-          var user;
-          var apiKey;
+          var data = {'auth': {'username': username, 'password': password}},
+              url = "/api/v1/auth/";
 
+          var posting = $.post( url, data );  
+
+          posting.done(function( data ) {
+          var content = $( data );
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: i would fix indentation -  it doesn't look like a function block

File: src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js [hunk 5]
```diff
@@ -44,30 +44,43 @@ $(document).ready(function() {
...
+          apiKey = content["0"]["token"]["api_key"];
+          userId = content["0"]["user"]["id"];
+          localStorage.setItem('auth_token', apiKey);
+          localStorage.setItem('user_id', userId);
+          $.post($('#loginform').attr('action'), {'_format': 'json', '_username': username, '_password': password}).done(function (done) {location.reload()});
+
+          });
+          // If posting is unsuccessful, then:
+          posting.fail(function(xhr) {
+            var content = $.parseJSON( xhr.responseText );
```
Comment (ahilles107) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: i would fix indentation - it doesn't look like a function block

Comment (ljubar) [inline src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/public/js/scripts/onload.js]: @ahilles107 indentation fixed

**Pull request merged**